Adding contributing guide to the project

victor-heliomar opened the pull request:
In order to make easier to collaborate with the project I added a non-exhaustive contributing guide

Israel-Laguan commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,74 @@
+# Welcome to the i18n Translation Generator contributing guide
+
+Thank you for investing your time in contributing to our project! Any contribution you make will be reflected on docs.github.com ✨.
```
```suggestion
Thank you for investing your time in contributing to our project! Any contribution you make will be reflected on docs.github.com ✨.
```

Not sure if this is accurate? Maybe this statement is valid in another repo

victor-heliomar replied:
Thanks! My mistake

Israel-Laguan commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,74 @@
+
+## Getting started
+
+There are a lot of things that you could do to collaborate on open sources projects!. Will leave you some ideas about what could you do to help us to improve i18n translation generator
```
NIT
```suggestion
There are a lot of things that you could do to collaborate on open sources projects!. Will leave you some ideas about what could you do to help us to improve i18n translation generator.
```

Israel-Laguan commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,74 @@
+
+#### Help to solve the community doubts
+
+If there is any doubt on any issue and you think you can help to solve it, then feel free to help to clear the doubts throught the comments box of the issue selected, always following our [CODE OF CONDUCTS](https://github.com/victor-heliomar/i18n-translation-generator/blob/dev/CODE_OF_CONDUCT.md)
```
Maybe you can change the URL a little 
```suggestion
If there is any doubt on any issue and you think you can help to solve it, then feel free to help to clear the doubts throught the comments box of the issue selected, always following our [CODE OF CONDUCTS](https://github.com/victor-heliomar/i18n-translation-generator/CODE_OF_CONDUCT.md)
```

victor-heliomar replied:
Proposed link seems like is not valid
![image](https://github.com/victor-heliomar/i18n-translation-generator/assets/66505715/a3a4a6d2-ae5c-40e2-8204-c2a661b936b4)

victor-heliomar replied:
But fixed the typo and added final end on each paragraph.- Thanks!

Israel-Laguan commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,74 @@
+
+1. Fork the repository.
+2. Clone the repository forked.
+3. Install or update to Node.js, at the version specified in .npmrc file.
```
NIT
```suggestion
3. Install or update to Node.js, at the version specified in `.npmrc` file.
```

Israel-Laguan commented on CONTRIBUTING.md:
```diff
@@ -0,0 +1,74 @@
+  -  We are currently working on some projects related to i18n translation generator in order to improve the way you create i18n translations the most possible. Projects in mind are:
+    1. Landing Page to explain easily the idea to developers.
+    2. Documentation Page in order to improve the doc of the project and make more easier the use of it.
+    3. Visual Studio Code Extension in order of create a user interface for users that doesn't feels comfortable with CLI Commands. Feel free of collaborate in any of them! Any help is appreciate!
```
NIT:
Add the last sentence in another line
```suggestion
    3. Visual Studio Code Extension in order to create a user interface for users that doesn't feel comfortable with CLI Commands. 
    
Feel free to collaborate on any of them! We would appreciate any help!
```

victor-heliomar replied:
Thanks! Done!

Israel-Laguan commented:
I would suggest some text about "rules in this repo", for example:

- JS code style? What about using something like eslint/prettier and leaving the config files here?
- A list of DON'Ts, for example, "We don't allow political comments or nationalistic supremacy, racial or religious claims, or anything that can hurt people" or "Please keep PRs short, if you have more than 1000 LOC please consider separate in different PRs", etc.
- Some link about PR/code review/comments etiquette

victor-heliomar commented:
> I would suggest some text about "rules in this repo", for example:
> 
> * JS code style? What about using something like eslint/prettier and leaving the config files here?
> * A list of DON'Ts, for example, "We don't allow political comments or nationalistic supremacy, racial or religious claims, or anything that can hurt people" or "Please keep PRs short, if you have more than 1000 LOC please consider separate in different PRs", etc.
> * Some link about PR/code review/comments etiquette

Hi! thanks for help reviewing. I think config for eslint/prettier can be on another PR. I think it would be useful to leave config file's path here. Will take note about that to create the issue

About the DON'Ts list you mentioned something similar to code of conduct, I already added a file for that (I used a template of GitHub but sounds good to me) https://github.com/victor-heliomar/i18n-translation-generator/blob/dev/CODE_OF_CONDUCT.md

About "Some link about PR/code review/comments etiquette" sounds like it is more for the file of code of conduct, not sure of adding that here, in any case could add it on another PR

victor-heliomar commented:
Added issues for https://github.com/victor-heliomar/i18n-translation-generator/issues/30 and https://github.com/victor-heliomar/i18n-translation-generator/issues/29 and added the note that the config file path should be put in this file
